docs/doxygen.cmake

```diff
@@ -307,11 +307,11 @@ function(ivw_private_make_help)
         "docpage{2}=\"\\page docpage-\\1 \\2\""
     )
 
-    set(additional_flags_list 
-        "LAYOUT_FILE            = ${HARG_HTML_DIR}/layout.xml"
-        "HTML_STYLESHEET        = ${HARG_HTML_DIR}/stylesheet.css"
-        "HTML_HEADER            = ${HARG_HTML_DIR}/header.html"
-        "HTML_FOOTER            = ${HARG_HTML_DIR}/footer.html"
+    set(additional_flags_list
```

Comment:
Why hard code an not use the function argument?

Reply:
To be honest it didn't work and I didn't understand why. But apparently while having this changed I did fix the issue with it, because when I change it now it works :D